ppr-ui/src/components/parties/EditDebtor.vue

```diff
@@ -132,14 +132,13 @@
                 <label class="general-label">Address</label>
               </v-col>
             </v-row>
-            <!-- <base-address ref="regMailingAddress"
+            <base-address ref="regMailingAddress"
                     id="address-debtor"
-                    :address="currentDebtor.address"
+                    v-model="currentDebtor.address"
                     :editing="true"
                     :schema="addressSchema"
```

Comment:
I talked to kaine about the validation and he said it's okay if we come back to it later. I think passing in the schema will give validation that is broken so it's better not to pass it in so that the validation is ignored by the address component

Reply:
If I don't pass it in, will it get updated in the state model?

Reply:
it should default to null and skip all the schema things -- that's what we want for now right?